### DISCUSSION TITLE
There is an error when I run “npm run complie" sample chat with "Creating an app"

### DISCUSSION THREAD

**Discussion by Byungmin-Kim** [feathersjs/feathers#3441] [Q&A] [answered]
I'm looking into using feathersjs to develop a backend service. 
Thank you in advance. I may have a question for you in the future.

There is an error when I run “npm run complie" sample chat with "Creating an app"

### Error Message 

```
node_modules/@feathersjs/schema/lib/json-schema.d.ts:63:98 - error TS2694: Namespace '"/.../.../sample-chat-test/node_modules/json-schema-to-ts/lib/types/definitions/index"' has no exported member 'JSONSchema7'.

63 export declare const queryProperty: <T extends import("json-schema-to-ts/lib/types/definitions").JSONSchema7, X extends {
```

There is a package-lock.json below.
```
"node_modules/json-schema-to-ts": {
      "version": "3.0.1",
      "resolved": "https://registry.npmjs.org/json-schema-to-ts/-/json-schema-to-ts-3.0.1.tgz",
      "integrity": "sha512-ANphQxnKbzLWPeYDmdoci8C9g9ttpfMx8etTlJJ8UCEmNXH9jxGkn3AAbMe+lR4N5OG/01nYxPrDyugLdsRt+A==",
      "dependencies": {
        "@babel/runtime": "^7.18.3",
        "ts-algebra": "^1.2.2"
      },
      "engines": {
        "node": ">=16"
      }
    }
```
### Test Envirment
```
Nodejs v21.7.1
@feathersjs/schema": "^5.0.23"

"feathers": {
  "language": "ts",
  "packager": "npm",
  "database": "sqlite",
  "framework": "koa",
  "transports": [
    "rest",
    "websockets"
  ],
  "schema": "typebox"
},
```

### Reason(maybe)
There is some difference between json-schema-to-ts.v3.0.1 and json-schema-to-ts.v3.0.0
JSONSchema7 is deleted in  json-schema-to-ts.v3.0.1 . 
It was commited in json-schema-to-ts yesterday.

Could you please check it and fix a problem?

**Comment by daffl** [comment, marked as answer]
I just made a commend at https://github.com/ThomasAribart/json-schema-to-ts/pull/192#issuecomment-1989814450 in the upstream module.

**Comment by daffl** [comment]
This should be fixed now in v5.0.24
